tasks/README.md

```diff
@@ -61,6 +61,8 @@ The preferred style of the keys should be `snake_case`.
 
 **implementations**: A list of implementation files in preference order, along with an optional list of required features for that implementation to be suitable on a target. The available features are defined by the task runner, but task runners should define at least the `shell`, `powershell` and `puppet-agent` features.
 
+**files**: Array of file names specified as strings that include the module containing the file. File specifications inding with / will require the entire directory. Files can also be provided as `implementations` objects.
```

Comment:
`inding` -> `ending. I'd also quote the slash `/`.

Reply:
We should clarify that only module directories that Puppet makes available via mount points will be accessible: files, lib, tasks.

Reply:
The files as implementation objects will require some more clarification. I imagine a use case for adding them in the implementations section would be to use the "requirement" key for making decisions based on what resources are available on the target. Will work on an example to illustrate.